python_modules/libraries/dagster-dbt/dagster_dbt_tests/cloud_v2/test_reconstruction.py

```diff
@@ -0,0 +1,49 @@
+import responses
+from dagster._core.code_pointer import CodePointer
+from dagster._core.definitions.definitions_class import Definitions
+from dagster._core.definitions.reconstruct import (
+    initialize_repository_def_from_pointer,
+    reconstruct_repository_def_from_pointer,
+)
+from dagster._utils.test.definitions import lazy_definitions
+
+from dagster_dbt_tests.cloud_v2.conftest import TEST_WORKSPACE
+
+
+@lazy_definitions
+def cacheable_dbt_cloud_workspace_data():
+    TEST_WORKSPACE.fetch_workspace_data()
+
+    return Definitions(
+        resources={"dbt_cloud": TEST_WORKSPACE},
+    )
+
+
+def test_cacheable_dbt_cloud_workspace_data(
```

Comment:
Test feels a bit over complicated, I don't feel like airlift tests needed to call out to another file? But assertions seem reasonable so won't block on that